source/docs/3/digitizing_basics.rst

```diff
@@ -40,287 +40,240 @@ Data Source [LINZ]_
+    :align: center
+   
+10. When creating a GIS layer, you must decide on each feature's attributes. Since this is a road layer, we additionally will have two primary attributes - Name and Class. In :guilabel:`New Field` Enter ``Name`` of the type :guilabel:`Text data,` with ``50`` as :guilabel:`Maximum length` and click :guilabel:`Add to attribute list.` Now create a new attribute ``Class`` of the type :guilabel:`Text data`, with ``50`` as :guilabel:`Maximum length`. Click :guilabel:`OK`
+
+ .. image:: /static/3/digitizing_basics/images/10.png
+    :align: center       
+	
+11. Once the ``Roads`` layer is loaded, click the :guilabel:`Toggle Editing` button to put the layer in editing mode.
+
+ .. image:: /static/3/digitizing_basics/images/11.png
+   :align: center
+   
+12. Click the :guilabel:`Add Line Feature` button. Click on the map canvas to add a new vertex. Add new vertices along with the road feature. Once you have digitized a road segment, right-click to end the feature.
+
+ .. image:: /static/3/digitizing_basics/images/12.png
+    :align: center
+	
 .. note::
 
    You can use the scroll wheel of the mouse to zoom in or out while digitizing.
    You can also hold the scroll button and move the mouse to pan around.
+   
 
-.. image:: /static/3/digitizing_basics/images/10.png
-   :align: center
-
-11. After you right-click to end the feature, you will get a pop-up dialog
-    called :guilabel:`Road - Feature Attributes`. Here you can enter attributes of the newly
-    created feature. Since the **fid** is an Autogenerate field, you
-    will not be able to enter a value manually. Leave it as such and enter the
-    road name as it appears on the topo map. Optionally, assign a Road Class
-    value as well. Click :guilabel:`OK`.
-
-.. image:: /static/3/digitizing_basics/images/11.png
-   :align: center
-
-12. The default style of the new line layer is a thin line. Let's change it so
-    we can better see the digitized features on the canvas. Right-click the
-    ``Roads`` layer and select :guilabel:`Properties`.
-
-.. image:: /static/3/digitizing_basics/images/12.png
-   :align: center
-
-13. Select the :guilabel:`Symbology` tab in the :guilabel:`Layer Properties`
-    dialog. Choose a thicker line style such as :guilabel:`topo road` from the
-    predefined styles. Click :guilabel:`OK`.
-
-.. image:: /static/3/digitizing_basics/images/13.png
-   :align: center
-
-14. Now you will see the digitized road feature clearly. Click :guilabel:`Save
-    Layer Edits` to commit the new feature to disk.
+   
+13. After you right-click to end the feature, you will get a pop-up dialog called :guilabel:`Road - Feature Attributes`. Here you can enter attributes of the newly created feature. Since the **fid** is an Autogenerate field, you will not be able to enter a value manually. Leave it as such and enter the road name as it appears on the topo map. Optionally, assign a Road Class value as well. Click :guilabel:`OK`.

```

Comment:
Skip entering any value for fid as it is a sequential id that will be autogenerated

Reply:
Done.